pom.xml

```diff
@@ -147,6 +147,7 @@
                     <artifactId>maven-failsafe-plugin</artifactId>
                     <version>3.0.0-M5</version>
                     <configuration>
+                        <useModulePath>false</useModulePath>
```

Comment:
Can you add a comment explaining this setting and referring to https://issues.apache.org/jira/browse/SUREFIRE-1831.

Reply:
Thanks! Comment added.